Translation corrections (#237)
07th-mod/higurashi-console-arcs: Update/yoig_tips_004.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 5, "ps3/s20/05/vtt_rika3014", 256, TRUE);
	OutputLine(NULL, "「早く済ませたほうがいいわ。",

> 			NULL, "\"You'd be best bringing a swift end to this.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"You need to end this quickly.", Line_WaitForInput);
```

//rあの日にいなくなった彼女が、どうして私の前に現れたのか、ということを…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "あの日にいなくなった彼女が、どうして私の前に現れたのか、ということを…。",

> 			NULL, "The reason why she who had died on that day appeared before me...", Line_Normal);

Reviewer: ```suggestion
			NULL, "The reason why the girl who died on that day appeared before me...", Line_Normal);
```
